modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -94,6 +101,62 @@ asmctl api create
 The API that was created is displayed.
 . Optionally, you can verify that an asset is created on Exchange and an API is created on API Manager.
 
+[[create-api-ASM-CLI-ConnAps]]
+== Create an API Using Anypoint Service Mesh With Connected Apps
+
+Use the procedure in this section to automatically create an API using Anypoint Service Mesh with your Connected App credentials. Before you create the API, you must complete the task prerequisite.
+
+*Task Prerequisite*
+
+Before you begin, ensure that:
+
+* You xref:obtain-connected-apps-credentials.adoc[Obtain the client ID and client secret credentials] from Connected Apps
+* The client credentials has the:
+** Exchange Contributor permission to publish the API to Exchange
+** Manage API Configuration to manage the API in API Manager
+
+To create an API using Anypoint Service Mesh with Connected Apps, perform the following steps:
+
+. Using Anypoint Service Mesh, type the following command to create the API:
++
+[source,text,linenums]
+----
+asmctl api create
+--name=_Kubernetes API resource name_ \
+--namespace=_namespace name_ \
+--environmentId=_environment ID_ \
+--groupId=_(Optional) group ID of the asset, for example, organization ID_ \
+--assetId=_asset ID on Exchange_ \
+--version=_asset version of the instance on exchange_ \
+--instanceLabel=_label of the instance on API Manager_ \
+--specTags=_list of comma separated tags to be added to the asset on Exchange_ \
+--instanceTags=_list of comma separated tags to be added to the instance on API Manager_ \
+--clientId=_ID of the Anypoint Platform Connected Apps, using which to create the API instance_ \
+--clientSecret=_client secret of the Anypoint Platform Connected Apps, using which to create the API instance_
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required or Optional | Default Value
+| `name` | The name of the Kubernetes API resource that you want to provision, for example, `orders-api`. | Required | Not applicable
+| `namespace` | The namespace in which you want to create the adapter, for example, `mulesoft-apps`. | Optional | default
+| `environmentId` | The ID of the environment where you want to create your API instance. | Required | Not applicable
+| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on the API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment. 
```

Comment:
```suggestion
| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID, which you obtain on the API Manager landing page or in Access Management | Optional | Organization ID of the organization that contains the environment. 
```
Should having this ID actually be a prerequisite, so that we don't send the audience off searching for it right in the middle of this task?

Reply:
Good point! Done! Thanks!